server/cli/launch.py

```diff
@@ -25,8 +24,8 @@
 @click.option(
     "--layout",
     "-l",
-    type=click.Choice(MODES),
-    default="umap",
+    default=[],
+    multiple=True,
     show_default=True,
     help="Method for layout."
```

Comment:
Can we add examples from MODES to the help text?

Reply:
This points out a confusing issue.   The argument takes 'layout_name', e,g 'umap', but the data must contain 'X_layout_name', eg, 'X_umap'

i wonder if the CLI shouldn't just require the literal variable name, eg, 'X_umap'

It would be less confusing IMHO.

Reply:
Message looks good. I see the discussion happening on slack as far as including the X_. I'll trust @sidneymbell  on this.

Reply:
@sidneymbell did not like this approach, so I have reverted and changed the help text to `"Layout name, eg, 'umap'."`

Reply:
That works, too. I don't have a good instinct as to which is more informative and less confusing.

Reply:
ditto.   so I'm going simple, and we can refine from here....